features/step-definitions/sqlup-mode.el-steps.el

```diff
@@ -51,3 +51,8 @@
         (cl-letf (((symbol-function 'set-process-sentinel) #'ignore)
                   ((symbol-function 'get-buffer-process) #'ignore))
           (sql-interactive-mode))))
+
+(When "I type a newline"
+      "Simulate typing a newline character"
+      (lambda ()
+        (execute-kbd-macro (string-to-vector "\n"))))
```

Comment:
Seems like it's impossible to do so I've rewritten the pull-request to instead fix the underlying issue that was causing the post command hook to error when run from a keyboard macro.